scripts/migrations/2020-08-update_migration_created_date.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+import json
+from common import get_aws_resource, scan_table
+
+READ_ONLY_ROLE_ARN = "arn:aws:iam::975596993436:role/storage-read_only"
+dynamodb = get_aws_resource("dynamodb", role_arn=READ_ONLY_ROLE_ARN)
+s3 = get_aws_resource("s3", role_arn=READ_ONLY_ROLE_ARN)
+
+vhs_table = "vhs-storage-staging-manifests-2020-07-24"
+
+backfill_vhs_table = "vhs-storage-staging-manifests-2020-08-19"
+
+
+def get_bucket_key(item):
+    try:
+        bucket = item["payload"]["namespace"]
+        key = item["payload"]["path"]
+    except KeyError:
+        try:
+            bucket = item["payload"]["bucket"]
+            key = item["payload"]["key"]
+        except:
+            raise RuntimeError(f"Find s3 bucket, key in {item}")
+        else:
+            return bucket, key
+    else:
+        return bucket, key
+
+
+def get_vhs_json(bucket, key):
+    return json.loads(s3.Object(bucket, key).get()["Body"].read().decode("utf-8"))
+
+
+def put_vhs_json(bucket, key, content):
+    try:
+        s3object = s3.Object(bucket, key)
+        # s3object.put(
+        #     Body=(bytes(json.dumps(content).encode('UTF-8')))
+        # )
+    except Error as e:
+        raise RuntimeError(f"Error updating backfill vhs object {bucket} {key}: {e}")
```

Comment:
If I've read correctly, this will overwrite the existing S3 object?

When I've done migrations in the past, I written to a brand new object – that way, the before/after objects are both preserved. It's not useful long-term, but it avoids blatting information, e.g. if I muck up the script and accidentally write empty data to S3.

Reply:
ok, will do. But if we update the pointer isn't it kind of the same thing? The old object will be lost anyway without anything referring to it won't it?